src/app/onprem/license-nag-message-management/nag-message.styl

```diff
@@ -0,0 +1,12 @@
+.nag-message
+  top 0
+  left 0
+  right 0
+  position fixed
+  z-index 16777271
+  line-height 30px
+  text-align center
+  background-color #FAC090
```

Comment:
I mean maybe it should go to https://github.com/codenvy/dashboard/blob/master/src/app/index.styl as a "root color" so we don't need to go in each component to see colors that are used
and then yes, it's a good way to use like more "generic name" like warning-color  and then use it inside the nag message